File: URI/1 - INICIANTE/C/2003 - DomingoDeManha.c
#include <stdio.h>

int main(){
	int h1, h2;
	while(scanf(" %d", &h1) == 1){
		scanf(":%d", &h2);
		h2 += 60;
		if(h2 > 60){
			h2 -= 60;
			h1 += 1;
		}
		if(h1 > 8){
			h2 += (h1-8) * 60;
			h1 -= h1-8;
		}
		if (h1 == 8){
			printf("Atraso maximo: %d\n", h2);
		}else{
			printf("Atraso maximo: 0\n");
		}
		
	}
	
	return 0;
}

File: URI/1 - INICIANTE/Python/2003 - DomingoDeManha.py
while True:
    try:
        hora = input().split(':')
        hora[1] = int(hora[1])
        hora[0] = int(hora[0])
        hora[1] += 60
        if hora[1] > 60:
            hora[1] -= 60
            hora[0] += 1
        if hora[0] > 8:
            hora[1] += (hora[0] - 8) * 60
            hora[0] -= hora[0] - 8
        if hora[0] == 8:
            print('Atraso maximo: {}'.format(hora[1]))
        else:
            print('Atraso maximo: 0')
    except EOFError:
        break